### DISCUSSION TITLE
how do i get rid of these "serialising behind stopper" errors?

### DISCUSSION THREAD

**Discussion by Amleexm1010** [gildor2/UEViewer#232] [Q&A]
sometimes when i try to open/export certain things this error comes up: 
Serializing behind stopper (6CD9+4 > 6CD9)
FPakFile::Serialize: file=/Game/Character/Common_NPCs/GenericFemaleNPC/NPCFemale_Hair_Jackie.uasset <- FArchive::ByteOrderSerialize <- TArray::Serialize: 908/16842752 <- FStaticMeshLODModel4<< <- TArray::Serialize: 0/2 <- UStaticMesh4::Serialize <- LoadObject: StaticMesh4'NPCFemale_Hair_Jackie.NPCFemale_Hair_Jackie', pos=6CD9, ver=505/0 (unversioned), game=ue4.13 <- UObject::EndLoad <- LoadWholePackage: /Game/Character/Common_NPCs/GenericFemaleNPC/NPCFemale_Hair_Jackie.uasset <- CUmodelApp::ShowPackageUI <- CUmodelApp::ProcessKey: key=6F, down=1 <- VisualizerLoop <- Main: umodel_build=1544

how do i stop this error?

**Comment by gildor2** [comment]
Check the forum thread about the game you're trying to open.
